Sources/Networking/AlamofireNetworkClient/AlamofireNetworkClient.swift

```diff
@@ -193,9 +193,18 @@ public extension AlamofireNetworkClient {
 
 // MARK: - Models
 public extension AlamofireNetworkClient {
-  enum Error: Swift.Error {
+  enum Error: LocalizedError {
     case request(RequestError, ErrorInfo)
     case other(Swift.Error, ErrorInfo)
+    
+    public var errorDescription: String? {
+      switch self {
+      case .request(let err, _):
+        return err.localizedDescription
+      case .other(let err, _):
+        return err.localizedDescription
+      }
+    }
```

Comment:
Maybe we can include `ErrorInfo` into the localized description?

Reply:
The idea is that `errorDescription` returns localized description from a concrete error that was handled by the caller. Here is an example...

```swift
AlamofireNetworkClient(session: session,
                       eventMonitors: [ANCSentryErrorMonitor()]) { origError, _ in
  CustomError(
    code: 1,
    message: "My custom error message",
    detail: origError.localizedDescription
  )
}
```

This is an implementation with the default error handler in place. So each error emitted will be transformed into CustomError. You see where I'm going...

When you get error from the API, you can now just write `error.localizedDescription` and it will return "My custom error message".

You can always cast this "Error" to AlamofireNetworkClient.Error to get out `ErrorInfo` or other info as well.